docs/guides/3-getting-started.md

```diff
@@ -16,6 +16,12 @@ First - you'll need the Ember CLI.  Head over to
 [Getting Started with Ember CLI](http://www.ember-cli.com/#getting-started) for 
 instructions.
 
+**TL;DR**
+
+```bash
+npm install -g ember-cli
+```
+
```

Comment:
First - you'll need the Ember CLI.  You can install it like so:

```bash
npm install -g ember-cli
```

Head over to [Ember CLI](http://www.ember-cli.com/) for more details.